tests/e2e/IncompatiblePhpParser/expected-output.txt

```diff
@@ -0,0 +1,10 @@
+Total: 1
```

Comment:
I am a bit confused because at the moment, if I `cd` to this directory and run `../../../build/infection.phar`, it works just fine...

Am I missing something?

Reply:
add this and the issue will be reproduced:

```diff
+/**
+ * @infection-ignore-all
+ */
+ class AstSampleProvide
```

Reply:
the key thing is to trigger execution of this line

https://github.com/infection/infection/blob/a663d6cecee222e9b33f11608073c134eb5e6916/src/PhpParser/Visitor/NonMutableNodesIgnorerVisitor.php#L59

Reply:
That did it, I was using the wrong Infection PHAR for some reason